src/testInteg/sam.test.ts

```diff
@@ -55,15 +55,6 @@ interface TestScenario {
 // to reduce the chance of automated tests timing out.
 const scenarios: TestScenario[] = [
     // zips
-    {
-        runtime: 'nodejs14.x',
```

Comment:
I see that node14 is deprecated now, this makes sense to delete right?

Reply:
> Do we need to update anything in `@aws-sdk` in npm?

not directly related